src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -98,6 +98,34 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
         return Promise.reject(new OrderFinalizationNotRequiredError());
     }
 
+    protected getGooglePayOptions(): GooglePayPaymentInitializeOptions {
+        return this._googlePayOptions as GooglePayPaymentInitializeOptions;
```

Comment:
avoid overriding types, as this could lead to bugs.

Reply:
method removed, instead, using superclass attributes as protected